Lens/Compiler/Context.Import.cs

```diff
@@ -91,6 +87,17 @@ public void ImportProperty<T>(string name, Func<T> getter, Action<T> setter = nu
 
         #region Helpers
 
+        /// <summary>
+        /// Throws an error if the assembly is marked as requiring save-to-disk.
+        /// </summary>
+        private void EnsureNotSaving()
```

Comment:
Would it be better to mark it as `[Conditional("NET_CLASSIC")]`?

Reply:
Wouldn't work, since `Conditional` requires the body to compile, but in `NET_CORE` the `AllowSave` does not exist.

Reply:
Ah, sorry, that's true. Okay then!